sim/scenarios/nat/nat-error-model.h

```diff
@@ -0,0 +1,42 @@
+#ifndef NAT_ERROR_MODEL_H
+#define NAT_ERROR_MODEL_H
+
+#include "../helper/quic-packet.h"
+#include "ns3/error-model.h"
+#include "ns3/random-variable-stream.h"
+#include <unordered_map>
+
+using namespace ns3;
+
+// The NATErrorModel drops packets enumerated by the user. This model does
+// a simple packet count and drops the specified packets. Packet numbering
+// starts at 1.
+
+// A hash function used to hash a pair of any kind
+struct hash_pair {
```

Comment:
As far as I can see, we don't need this, right?
At least the code compiles without it.